### ISSUE TITLE
Fail: 'I' -> NN and not PRP

### ISSUE THREAD

**Issue by AmenRa666** [dariusk/pos-js#25]
I tried this script:
```
var words = new pos.Lexer().lex('I love NodeJS');
var tagger = new pos.Tagger();
var taggedWords = tagger.tag(words);
for (i in taggedWords) {
    var taggedWord = taggedWords[i];
    var word = taggedWord[0];
    var tag = taggedWord[1];
    console.log(word + " /" + tag);
}
```

This is the log:
```
I /NN
love /NN
NodeJS /NN
undefined /undefined
undefined /undefined
```

It fails with the pronoun 'I'.
I don't know why it print two undefined elements, no words remaining.

**Comment by Hugo-ter-Doest** [comment]
I tried your script and did not get the undefined/undefined results:
node scriptByElias.js 
I /NN
love /NN
NodeJS /NN

Regards, 
Hugo

**Comment by AmenRa666** [comment]
Thank you man!

**Comment by dariusk** [comment]
Thanks for this, just pushed a fix, publishing to NPM now, version `0.4.1`.
